src/props.supports.css

```diff
@@ -8,9 +8,16 @@
 :where(html) {
   --isLTR: 1;
   --isRTL: -1;
-  
+
   &:dir(rtl) {
+      /* PostCSS compiles this to [dir="rtl"] :where(html) for older browsers, which doesn't work */
+    --isLTR: -1;
+    --isRTL: 1;
+  }
+
+  &[dir="rtl"] {
+    /* Force the dir attribute selector to attach to :where(html) */
```

Comment:
this is a nice backup to using `:dir()` anyway, we'll do it ourselves in our own nesting way and not use the polyfill that's part of `postcss-preset-env`